Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -339,13 +363,15 @@ extension SwiftLanguageService {
   }
 
   public func documentDependenciesUpdated(_ uri: DocumentURI) async {
-    guard let snapshot = try? self.documentManager.latestSnapshot(uri) else {
-      return
+    // FIXME: Debounce, otherwise it will get called over and over
```

Comment:
Any reason not to debounce it now?

Reply:
Oh, this was just an in-development comment that I forgot to remove. We `fileDependenciesUpdated` on the `BuildSystem` level now, so this isn’t needed. I just put it here at some point to remind myself that we need to debounce somewhere.